test/sample.kt

```diff
@@ -287,7 +302,9 @@ class Derived() : Base() {
     override fun v() {}
 }
 
-open class AnotherDerived() : Base() {
+open class AnotherDerived()
+    : Base()
```

Comment:
i wonder if changes like this should be configurable. I'm not very in touch with what "standard" kotlin looks like these days so I'm not sure what is "correct" here. My gut says not this, but I suspect at the very least opinions will differ. Same thoughts on line 219.